vnext/ReactUWP/Modules/Animated/NativeAnimatedNodeManager.cpp

```diff
@@ -164,7 +164,7 @@ void NativeAnimatedNodeManager::StartTrackingAnimatedNode(
           updatedAnimationConfig.insert(
               static_cast<folly::StringPiece>(s_framesName), [animationConfig, activeFrames = item.second->Frames()]() {
                 auto frames = folly::dynamic::array();
-                for (auto const &frame : animationConfig.find("frames").dereference().second) {
+                for ([[maybe_unused]] auto const &frame : animationConfig.find("frames").dereference().second) {
```

Comment:
>&frame [](start = 49, length = 6)

Can't we just do a simple for (unsigned int i=0; i<animationConfig.find("frames").dereference().second.length; i++){frames.push_back(0.0);}
?

Reply:
We probably could, but I do not know the underlying implementation details and how to test that change.

---
In reply to: [370871461](https://github.com/microsoft/react-native-windows/pull/3958#discussion_r370871461) [](ancestors = 370871461)